src/Raven.Server/Documents/Replication/ResolveConflictOnReplicationConfigurationChange.cs

```diff
@@ -105,56 +128,29 @@ public void RunConflictResolversOnce()
                     _log.Info("Failed to run automatic conflict resolution", e);
             }
         }
-
-
-        private class ResolveConflictsCommand : TransactionOperationsMerger.MergedTransactionCommand
+        
+        private class PutResolvedConflictsCommand : TransactionOperationsMerger.MergedTransactionCommand
         {
-            private readonly List<DocumentConflict> _conflicts;
+            private readonly List<DocumentConflict> _resolvedConflicts;
             private readonly ResolveConflictOnReplicationConfigurationChange _resolver;
 
-            public ResolveConflictsCommand(List<DocumentConflict> conflicts, ResolveConflictOnReplicationConfigurationChange resolver)
+            public PutResolvedConflictsCommand(List<DocumentConflict> resolvedConflicts, ResolveConflictOnReplicationConfigurationChange resolver)
```

Comment:
It shouldn't. Consider the case that you read the conflict at T1, then write it in T3.
But in T2, it was resolved externally.